source/guide_lychee.rst

```diff
@@ -0,0 +1,88 @@
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/lychee.png 
+      :align: center
+
+#########
+Lychee
+#########
+
+Lychee_ is a open source photo-management software written in PHP and distributed under the MIT license. It allows you to easily upload, sort and manage your photos, all while presenting those with a beautiful web interface.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * PHP_
+  * MySQL_ 
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Also, the domain you want to use for Lychee must be set up as well:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your `document root`_, then clone the latest release of the Lychee installer and move the files to the document root:
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ git clone https://github.com/electerious/Lychee.git
+ Cloning into 'Lychee'...
+ remote: Counting objects: 10458, done.
+ remote: Compressing objects: 100% (8/8), done.
+ remote: Total 10458 (delta 3), reused 7 (delta 2), pack-reused 10448
+ Receiving objects: 100% (10458/10458), 5.74 MiB | 8.85 MiB/s, done.
+ Resolving deltas: 100% (7144/7144), done.
+ [isabell@stardust html]$ mv Lychee/* Lychee/.[!.]* . && rmdir Lychee
```

Comment:
Please do this step by step. We're not in a hurry. Also I don't think `.[!.]*` is self explanatory.